src/XIVLauncher.Common.Unix/Compatibility/CompatibilityTools.cs

```diff
@@ -57,7 +57,14 @@ public CompatibilityTools(WineSettings wineSettings, Dxvk.DxvkHudType hudType, b
         this.toolDirectory = new DirectoryInfo(Path.Combine(toolsFolder.FullName, "beta"));
         this.dxvkDirectory = new DirectoryInfo(Path.Combine(toolsFolder.FullName, "dxvk"));
 
-        this.logWriter = new StreamWriter(wineSettings.LogFile.FullName);
+        try
+        {
+            this.logWriter = new StreamWriter(wineSettings.LogFile.FullName);
+        }
+        catch (IOException e)
+        {
+            // For some reason, this is called on Windows, and then it crashes xlcore because wine.log is considered open by another process already.
```

Comment:
Yeah I also noticed that when testing repair on windows and made sure it doesn’t happen here https://github.com/goatcorp/FFXIVQuickLauncher/blob/0e7473a271ddb41a7b763fafaf5c64a0a033bf18/src/XIVLauncher.Core/Components/SettingsPage/Tabs/SettingsTabWine.cs#L111

Probably even cleaner would be to not save settings pages that are hidden